spec/support/generators.rb

```diff
@@ -0,0 +1,22 @@
+module Pageflow
+  module Specs
+    module Generators
+      module Macros
+        def setup_default_destination
+          # Tell the generator where to put its output (what it thinks of as Rails.root)
+          destination File.expand_path('../../tmp', __FILE__)
+
+          before { prepare_destination }
+        end
+      end
+
+      def self.included(klass)
+        klass.extend(Macros)
+      end
```

Comment:
I believe that style has fallen out of fashion; it's too magical. These days the RSpec dogma is that explicit is better. But since the rest of the code is like that, I'll do it that way.